spec/features/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+feature "user deletes review" do
+  scenario "user adds a review successfully and deletes successfully" do
+    create_zoo
+    write_review
+    click_link "Delete Review"
+    expect(page).to have_content "Review deleted successfully"
+    expect(page).to have_content "Franklin Park Zoo"
+  end
+
+  scenario "user that didn't create review can't delete review" do
+    create_zoo
```

Comment:
this should be a before statement since you're using create_zoo & write_review in both scenarios!